app/config/temple/events.json

```diff
@@ -1,51 +1,43 @@
 {
     "events": [
         {
-            "name": "Knowing Our History: Constructing Identity & Building Pride. Led by: Chalice Blythe",
+            "name": "Topic: The Power of Naming Yourself. Led by: Minister Hadrian Flyte,
```

Comment:
You dropped a quote here. interesting that the checks didn't catch it

Reply:
@morganecf how hard would it be to add frontend tests to catch this sort of thing? like, i am pretty sure this would generate a react error

Reply:
though i honestly don't understand why it doesn't break at the protoc level. maybe cuz that's only for rooms?...